legal/src/main/resources/legal/NOTICE.md

```diff
@@ -64,11 +64,11 @@ http://www.apache.org/licenses/LICENSE-2.0.html.
 
```

Comment:
When editing this file already, the year in the `Copyright Holders` section could be updated.